server/datastore/mysql/migrations/tables/20240313085226_AddCalendarEventTables.go

```diff
@@ -0,0 +1,52 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20240313085226, Down_20240313085226)
+}
+
+func Up_20240313085226(tx *sql.Tx) error {
+	// TODO(lucas): Check if we need more indexes.
+
+	if _, err := tx.Exec(`
+	CREATE TABLE IF NOT EXISTS calendar_events (
+		id INT(10) UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  		email VARCHAR(255) NOT NULL,
+		start_time TIMESTAMP NOT NULL,
+		end_time TIMESTAMP NOT NULL,
+		event JSON NOT NULL,
+
+		created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+		updated_at TIMESTAMP NOT NULL NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP
+	);
+`); err != nil {
+		return fmt.Errorf("create calendar_events table: %w", err)
+	}
+
+	if _, err := tx.Exec(`
+	CREATE TABLE IF NOT EXISTS host_calendar_events (
+		id INT(10) UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+		host_id INT(10) UNSIGNED NOT NULL,
```

Comment:
DELETE CASCADE if host entry is deleted?

Reply:
We usually not link the tables the `hosts` (to avoid performance issues) and do the cleanup manually, see:

https://github.com/fleetdm/fleet/blob/2522cc5ef6bdf71c35c324765c1ef96b4606f238/server/datastore/mysql/hosts.go#L479-L505

I'll add it now.